src/components/molecules/StartTable/StartTable.scss

```diff
@@ -0,0 +1,44 @@
+@import "scss/constants.scss";
+
+$jazzbar-table-height: 189px;
+
+.StartTable {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  background: $dark;
+  border-radius: 22px;
```

Comment:
Please check if this value exists in the `constants.scss`

Reply:
there is no value `22px` in the `constants.scss`